src/utils/referral.js

```diff
@@ -79,36 +81,188 @@ const echisNHDDValuesCoding = {
     code: `29541`,
     display: `Lack of sleep`
   },
-  feeling_of_worthlessness: {
-    system: SNOMED_CT_URL,
-    code: `247892001`,
-    display: `Feeling of worthlessness`
-  },
+  // feeling_of_worthlessness: {
```

Comment:
This is required for [PNC Mother](https://github.com/moh-kenya/config-echis-2.0/blob/main/forms/app/postnatal_care_service.xlsx), i believe it's part of mental danger signs. Some of the mappings commented out might actually be needed